build-system/tasks/karma.conf.js

```diff
@@ -23,6 +23,8 @@ const globby = require('globby');
 const {gitCommitterEmail} = require('../common/git');
 const {isTravisBuild, travisJobNumber} = require('../common/travis');
 
+const argv = require('minimist')(process.argv.slice(2));
```

Comment:
nit: Move to line 18 so it can live with the other require statements.

Reply:
done.